janus_server/src/datastore.rs

```diff
@@ -88,11 +88,17 @@ impl Transaction<'_> {
             .prepare_cached(
                 "INSERT INTO tasks (id, ord, aggregator_endpoints, vdaf, vdaf_verify_param,
                 max_batch_lifetime, min_batch_size, min_batch_duration, collector_hpke_config)
-                VALUES ($1, 0, '{}', 'PRIO3', '', 0, 0, INTERVAL '0', '')",
+                VALUES ($1, 0, '{}', $2, '', 0, 0, INTERVAL '0', '')",
             )
             .await?;
         self.tx
-            .execute(&stmt, &[/* task_id */ &&task.id.0[..]])
+            .execute(
+                &stmt,
+                &[
+                    &&task.id.0[..],         // id
+                    &Vdaf::Prio3Aes128Count, // vdaf
```

Comment:
Surely this should use `task._vdaf` (and of course take the opportunity to rename the field to `vdaf` since it will now be read). Also the tests should be updated to verify that the expected `vdaf_identifier` value is written.